code/game/objects/items/food/meat.dm

```diff
@@ -575,7 +576,7 @@
 	name = "double rat-kebab"
 	icon_state = "doubleratkebab"
 	tastes = list("rat meat" = 2, "metal" = 1)
-	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 16, /datum/reagent/consumable/nutriment/vitamin = 6)
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 20, /datum/reagent/consumable/nutriment/vitamin = 4)
```

Comment:
this is not how food code works, now its way more than double.

Reply:
i was told from up high to make the numbers 2x as big so i did that

Reply:
@Qustinnus can you elaborate because i dont know what you mean

Reply:
nvm im wrong, i thought this was the food thats crafted from the regular rat kebab, which would've made it add reagents ontop of the original reagents.